pinecone/client_test.go

```diff
@@ -11,14 +12,18 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/google/go-cmp/cmp"
+	"github.com/pinecone-io/go-pinecone/internal/gen/control"
+
```

Comment:
nit: These imports seem unordered 🤔 did you run `gofmt`?

Reply:
I did! But I'll do it again :)

Reply:
[we spoke offline & this is actually correct; leaving as-is]